storage/client/src/main/java/com/azure/storage/blob/BlockBlobAsyncClient.java

```diff
@@ -47,6 +63,7 @@
  * object through {@link Mono#toFuture()}.
  */
 public final class BlockBlobAsyncClient extends BlobAsyncClient {
+    static final long BLOB_DEFAULT_UPLOAD_BLOCK_SIZE = 4 * Constants.MB;
```

Comment:
This should be public. We should also have a constant BLOB_MAX_UPLOAD_BLOCK_SIZE = 100*Constants.MB

Reply:
Added and checked in uploadFromFile.